source/release-notes/4.4.txt

```diff
@@ -1915,6 +1915,18 @@ output:
   - :serverstatus:`shardedIndexConsistency`
   - :serverstatus:`shardingStatistics.rangeDeleterTasks`
   - :serverstatus:`shardingStatistics.unfinishedMigrationFromPreviousPrimary`
+  - :serverstatus:`shardingStatistics.numHostsTargeted`
+
+``serverStatus`` Sharding Statistics Output Change
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+CRUD Operations and Aggregations
```

Comment:
I'd remove this line. It's kind of trying to act as a heading but looks a little strange in plain-text. The copy reads fine and still makes sense without this line.

Reply:
Updated.